README.md

```diff
@@ -25,10 +25,10 @@ req('foo').then(…)
 req('foo').then(…)
 ```
 
-## SEE ALSO
```

Comment:
I like the tests, but as a rule I use `tap` for my tests, not `jest`. Ordinarily I'm not that picky, but as I'll have to maintain this going forward, I would like to see them in tap format before landing.

Reply:
Sorry, I didn't know about your preference. Maybe I should have checked beforehand. But fair point, if you are going to maintain it, it should be in tooling that you feel comfortable with.
That being said, we are not using this module in production anymore and I hardly find extra time to contribute, so we can either close it or leave it for the community to take over and apply those changes. Hope it's fine.